test/tap/tap/Makefile

```
@@ -37,10 +39,10 @@ default: all
 
 .PHONY: all
 all: libtap_mariadb.a libtap_mysql57.a libtap_mysql8.a \
-	libtap.so libcpp_dotenv.so libre2.so
+	libtap.a libtap.so libcpp_dotenv.so libre2.so
 
 debug: OPT := $(STDCPP) -O0 -DDEBUG -ggdb -Wl,--no-as-needed $(WASAN)
```

checkmake (0.2.2)

[warning] Target "debug" should be declared PHONY.

(phonydeclared)